src/ILCompiler.ReadyToRun/src/IBC/ibcdatamodel.cs

```diff
@@ -0,0 +1,537 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace ILCompiler.IBC
+{
+    public class IBCException : Exception
+    {
+        public IBCException(String message) : base(message) { }
+    }
+
+    public static class IBCData
+    {
+        static public SectionTypeInfo[] sectionTypeInfo;
+
+        // Methods and types for managing the various stream types
+
+        public enum TokenType
+        {
+            MetaDataToken,
+            MethodToken,
+            TypeToken,
+            TokenTypeOther
+        }
+
+        public struct SectionTypeInfo
+        {
+            public TokenType tokenType;
```

Comment:
Mark these two `readonly` and PascalCase?

Reply:
Done